ui/src/views/People/Search.vue

```diff
@@ -59,8 +77,28 @@ import axios from "axios";
 
 import Alert from "@/components/Layout/Alert.vue";
 import DynamicForm from "@/components/Form/DynamicForm.vue";
-
+import MobileLayout from "@/mixins/MobileLayout.js";
 export default {
+  mixins: [MobileLayout],
+  computed:{
+    setTitleStyle(){
```

Comment:
Same comment as earlier. You aren't setting anything here. This applies to all the computed properties.

Reply:
Done